stl/inc/format

```diff
@@ -756,6 +768,18 @@ public:
     }
 };
 
+// Fetch the value of an argument associated with a dynamic
+// width or precision specifier. This will be called with either
+// _Width_checker or _Precision_checker as "_Handler".

```

Comment:
A constraint would work. In general I prefer `_STL_INTERNAL_STATIC_ASSERT` for internal machinery since it has nearly no throughput cost outside of our test suite, but this would be a fairly cheap _`requires-clause`_.